packages/onboarding/src/navigator/navigator-buttons/index.tsx

```diff
@@ -22,10 +22,10 @@ interface NavigatorButtonAsItemProps extends NavigatorItemProps {
 	path: string;
 }
 
-export function NavigatorItem( { icon, checked, active, children, ...props }: NavigatorItemProps ) {
+export function NavigatorItem( { icon, active, children, ...props }: NavigatorItemProps ) {
```

Comment:
We also need to remove the `checked` prop from the type definition

Reply:
We cannot, since the check icon is still required in the `Select styles` screen.